apps/bare-expo/babel.config.js

```diff
@@ -1,18 +1,22 @@
 module.exports = function(api) {
   api.cache(true);
+
+  const moduleResolverConfig = {
+    alias: {
+      '~expo': 'expo',
+      expo: './moduleResolvers/expoResolver',
+    },
+  };
+
+  // On the CI we'd like to get rid of `native-component-list` being a part of the final bundle.
+  // Otherwise, some tests may fail due to timeouts (bundling takes significantly more time).
+  if (process.env.CI || process.env.NO_NCL) {
+    moduleResolverConfig.alias['^native-component-list(/.*)?'] = './moduleResolvers/nclResolver';
```

Comment:
Let's rename `nclResolver` to `nullResolver`. It will make more sense to have the explanation only in one place and treat `nullResolver` more as a utility, not another place where you could put some logic.